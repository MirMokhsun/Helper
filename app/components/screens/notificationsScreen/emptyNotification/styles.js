import { StyleSheet } from 'react-native';

export const styles = StyleSheet.create({
    container: {
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
    },
    text: {
        fontFamily: 'Roboto-Regular',
        fontSize: 20,
        lineHeight: 24,
        color: '#9FA0B2',
        marginTop: 15,
    },
});

