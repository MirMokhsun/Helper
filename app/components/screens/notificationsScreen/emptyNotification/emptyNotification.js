import React, { PureComponent } from 'react';
import { Text, View } from 'react-native';
import { styles } from './styles';
import EmptyNotificationSVG from '../../../commonComponents/svg/EmptyNotificationSVG';

export class EmptyNotification extends PureComponent {
    constructor(props) {
        super(props);
    }
    
    render() {
        const { text } = this.props;
        return (
            <View style={styles.container} >
                <EmptyNotificationSVG />
                <Text style={styles.text}>
                    {text}
                </Text>
            </View>
        );
    }
}   