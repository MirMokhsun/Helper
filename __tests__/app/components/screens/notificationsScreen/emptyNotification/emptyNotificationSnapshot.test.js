import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { EmptyNotification } from '../../../../../../app/components/screens/notificationsScreen/emptyNotification/emptyNotification';

describe('EmptyNotification', () => {
    test('Create snapshot EmptyNotification', () => {
        const renderer = new ShallowRenderer()
        let emptyNotification = renderer.render(<EmptyNotification text={'test'}/>);
        expect(emptyNotification).toMatchSnapshot();
    });
});