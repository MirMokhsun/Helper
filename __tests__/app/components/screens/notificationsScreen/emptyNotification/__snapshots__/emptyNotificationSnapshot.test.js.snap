// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyNotification Create snapshot EmptyNotification 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <EmptyNotificationSVG />
  <Text
    style={
      Object {
        "color": "#9FA0B2",
        "fontFamily": "Roboto-Regular",
        "fontSize": 20,
        "lineHeight": 24,
        "marginTop": 15,
      }
    }
  >
    test
  </Text>
</View>
`;
